backend/dep/wire.go

```diff
@@ -223,6 +224,7 @@ func InjectRoutingService(
 	dataDogAPIKey provider.DataDogAPIKey,
 	segmentAPIKey provider.SegmentAPIKey,
 	ipStackAPIKey provider.IPStackAPIKey,
+	searchTimeout provider.SearchAPITimeout,
```

Comment:
```suggestion
	searchTimeout provider.SearchTimeout,
```

Reply:
Search is our internal service. Let's put this parameter before the external API keys.